pom.xml

```diff
@@ -78,36 +78,29 @@ not datatype, data format, or JAX-RS provider modules.
   <build>
     <pluginManagement>
       <plugins>
-	<plugin>
-	  <!-- Inherited from oss-base. Generate PackageVersion.java.-->
+        <plugin>
+          <!-- Inherited from oss-base. Generate PackageVersion.java.-->
           <groupId>com.google.code.maven-replacer-plugin</groupId>
           <artifactId>replacer</artifactId>
-	  <executions>
+          <executions>
             <execution>
               <id>process-packageVersion</id>
               <phase>generate-sources</phase>
             </execution>
           </executions>
-	</plugin>
+        </plugin>
 
-	<plugin>
+        <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-surefire-plugin</artifactId>
           <configuration>
             <excludes>
               <exclude>com/fasterxml/jackson/**/failing/*.java</exclude>
             </excludes>
           </configuration>
-	</plugin>
+        </plugin>
       </plugins>
     </pluginManagement>
-
```

Comment:
Hmmh. Wait, what, this will remove module metadata from all components, not just Afterburner/Blackbird.
I don't think we want that as it should work fine for others (JAXB annotations, mr-bean, paranamer etC).

Reply:
Yes. Sorry that seems to be a misunderstanding. What I meant to say is, don't touch this part at all ins this PR as it is unrelated. (don't remove, don't change the version)

Reply:
I created #200 - you can assign that to me and I will follow up.

Here, you should roll back the change to the root 'pom.xml' completely before merging.

Reply:
* the only way to get the CI build to pass right now is to remove the gradle module plugin - the jdk17 build fails 100% of the time because of it
* we can add the gradle module plugin back when we work out how to get it work with the dependency-reduced-pom.xml

Reply:
By chance I just talked to someone with more Maven knowledge. We found this solution (I'll post more details later on the other issue). If you want to have this fixed properly in this PR already you can do this:

https://github.com/FasterXML/jackson-modules-base/issues/200#issuecomment-1416711529

(instead of removing the gradle metadata plugin completely in this repo)

Reply:
reverted this bit and applied @jjohannes fix